src/api/app/views/event_mailer/_submit_action.text.erb

```diff
@@ -1,4 +1,4 @@
 - <%= a['type'] %> <%= a['sourceproject'] %>/<%= a['sourcepackage'] %> => <%= a['targetproject'] %>/<%= a['targetpackage'] %>
 <% if a['diff'] -%>
-<%= a['diff'] %>
+<%= a['diff'].force_encoding('UTF-8') %>
```

Comment:
I wonder if this could be done when storing diff instead of when rendering :thinking:  @eduardoj did you check that?

Reply:
:doughnut:  :doughnut:  :doughnut:

The diff is retrieved from the backend [here](https://github.com/openSUSE/open-build-service/blob/175e8ced727229a02ad3abf2df78b0cf299dd698/src/api/app/models/bs_request_action.rb#L68).

:doughnut:  :doughnut:  :doughnut: